llvm/lib/Target/RISCV/RISCVInstrInfo.td

```diff
@@ -2017,6 +2169,74 @@ def : Pat<(binop_allwusers<add> GPR:$rs1, (AddiPair:$rs2)),
 }
 
 //===----------------------------------------------------------------------===//
+
```

Comment:
please add these to a new MIPS specific file like other vendor extensions.

Reply:
Addressed in https://github.com/llvm/llvm-project/pull/121394